src/main.cpp

```diff
@@ -233,8 +233,14 @@ void destroy() {
 		throw std::runtime_error("/usr/bin/tldr could not be removed");
 	}
 	std::cout << "Removed executable /usr/bin/tldr" << std::endl;
-
-	string cache_dir = "/home/" + string(su) + "/.tldr";
+	
+	string cache_dir = "";
+	if (string(su) == "root") {
+		cache_dir = "/root/.tldr";
```

Comment:
I think when someone is executing it as root, it's probably by accident. I'd say we should just cancel the operation then. 
And I would absolutely not recommend running my client as root in either case.